doc/sphinx-guides/source/installation/config.rst

```diff
@@ -918,8 +918,8 @@ Before going live, you might want to consider setting up integrations to make it
 
 .. _jvm-options:
 
-JVM Options
------------
+JVM (and other) Options
+-----------------------
```

Comment:
Hmm, I'm not in love with the idea of "and other". We risk this section being cluttered by a bunch of random stuff.

How about a new section called "Application Server Settings" or something?

Reply:
done!